core/src/main/scala/com/avast/grpc/jsonbridge/ReflectionGrpcJsonBridge.scala

```diff
@@ -103,9 +103,16 @@ object ReflectionGrpcJsonBridge extends StrictLogging {
     }
 
   private def createFutureStubCtor(sd: ServiceDescriptor, inProcessChannel: Channel): () => AbstractStub[_] = {
-    val serviceGeneratedClass = Class.forName {
-      if (sd.getName.startsWith("grpc.")) "io." + sd.getName + "Grpc" else sd.getName + "Grpc"
+    val serviceClassName = if (sd.getName.startsWith("grpc.")) {
```

Comment:
Please check if the `if` is still required after this change. We introduced this `if` to correctly map standard `io.grpc.protobuf.services.ProtoReflectionService`.

Reply:
TBH I don't know why the `if` is present. It just seemed to me like a good idea to not remove it :smile: If you understand the reason, could _you_ please check it?

Reply:
I found it. It **is* still needed to convert `grpc.reflection.v1alpha.ServerReflection` to `io.grpc.reflection.v1alpha.ServerReflectionGrpc` correctly.

Reply:
I take it back - I originally got you wrong. It's not needed anymore, I removed it.